run/h2c/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+
+	"golang.org/x/net/http2"
+)
+
+func main() {
```

Comment:
nit: Do we think side-by-side comparison with the "usual" way we start the server is important? If so there are a few syntactic differences that could be more aligned with the other samples.

Reply:
I don't think there's much value in going into too much detail. Likely people who know Go will refer to this, as every language will be vastly different in terms of how to support h2c.